<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>638</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>775</width>
    <height>617</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>975</width>
    <height>686</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="myname_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,6">
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true">QListWidget::item
{
height:40px; /*每个item的高度*/
font-size:20px; /*字体大小*/
}

QListWidget::item::hover
{
background-color:rgb(9,150,200,100); /*伪状态，鼠标经过背景色*/
}

QListWidget::item:selected
{
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc); /*选中背景色*/
}</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QScrollArea" name="scrollArea">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>643</width>
           <height>551</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="infoMenu">
    <property name="title">
     <string>信息</string>
    </property>
    <addaction name="refreshAction"/>
    <addaction name="myinfoAction"/>
   </widget>
   <widget class="QMenu" name="aboutMenu">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="infoMenu"/>
   <addaction name="aboutMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="refreshAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>refresh online user</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="myinfoAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>My info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
